integration/env/env.go

```diff
@@ -1,13 +1,14 @@
 // +build k8srequired
 
-package integration
+package env
```

Comment:
With all the separation there is now a separate package for this but it cannot be called `init` since this is reserved for the native init function.